test/integration/068_partial_parsing_tests/extra-files/ref_override2.sql

```diff
@@ -0,0 +1,4 @@
+- Macro to override ref xxxx
+{% macro ref(modelname) %}
```

Comment:
I've added a test to check that the special_override_macros are correct. We don't need to test each name individually because for the purposes of this code there are only two variations and both are tested.

Reply:
Ok checking that `special_override_macros` is what we expect is good enough for me.